workspace.json

```diff
@@ -4,8 +4,10 @@
     "apple-pay-integration": "packages/apple-pay-integration",
     "core": "packages/core",
     "checkout-button-integration": "packages/checkout-button-integration",
+    "e2e": "packages/e2e",
```

Comment:
The test files should live in the payment integration packages, e.g.: `google-pay`, not in a centralised package, so we can easily assign code owners.

Reply:
Moved test files into payment packages. The rules are set as below:

- `playwright` will run all tests inside a payment package's `e2e` folder.
- `jest` will ignore all `e2e` folders.